src/App.vue

```diff
@@ -38,10 +38,16 @@ export default {
   },
   data: () => ({
     isMobileApp: config.mobileApp,
+    networksThemes: ["cosmos", "terra", "emoney", "kava", "kusama"],
```

Comment:
I am a little worried, that this doesn't scale

Reply:
How to do it then? CSS is quite dumb

Reply:
OK, so what's the plan Stan

Reply:
I talked with @jbibla to experiment with default variables. We could just set default variables (lunie-dark) as a default and overwrite these variables with the network class

Reply:
Sounds good. Unsure how to implement that without a similar logic than what we have in this PR (array with current networks slugs)

Reply:
how about setting .lunie-dark on #app?

Reply:
just tried. It doesn't work